### PR TITLE
Bump models to v0.3.0

### PR DESCRIPTION
**Pull request by elazarg**

### CODE REVIEW

File: pkg/ibmvpc/nacl_analysis.go [hunk 1]
```diff
@@ -59,35 +60,37 @@ func getProperty(p *int64, defaultP int64) int64 {
...
-		conns = getTCPUDPConns(*ruleObj.Protocol,
-			getProperty(ruleObj.SourcePortMin, common.MinPort),
-			getProperty(ruleObj.SourcePortMax, common.MaxPort),
-			getProperty(ruleObj.DestinationPortMin, common.MinPort),
-			getProperty(ruleObj.DestinationPortMax, common.MaxPort),
+		protocol := netp.ProtocolStringUDP
+		if *ruleObj.Protocol == protocolTCP {
+			protocol = netp.ProtocolStringTCP
+		}
+		conns = connection.TCPorUDPConnection(protocol,
```
Comment (zivnevo) [inline pkg/ibmvpc/nacl_analysis.go:89]:
```suggestion
		conns = getTCPUDPConns(*ruleObj.Protocol,
```

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: cmd/analyzer/parse_args.go [hunk 2]
```diff
@@ -330,7 +329,7 @@ func invalidArgsExplainMode(args *InArgs, flagset *flag.FlagSet) error {
 	}
 
 	protocol := strings.ToUpper(*args.EProtocol)
-	if protocol != string(common.ProtocolTCP) && protocol != string(common.ProtocolUDP) && protocol != string(common.ProtocolICMP) {
+	if protocol != "TCP" && protocol != "UDP" && protocol != "ICMP" {
```
[the remark this answers is not in the document]
Comment (elazarg) [reply cmd/analyzer/parse_args.go:333]: I did not want to leak the library to the user.

File: pkg/ibmvpc/nacl_analysis.go (hunk 1 above)
Comment (elazarg) [reply pkg/ibmvpc/nacl_analysis.go]: Done

File: cmd/analyzer/parse_args.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (elazarg) [reply cmd/analyzer/parse_args.go]: Done

**Review by zivnevo — APPROVED** [review]

**Pull request merged**